subjects/slice/README.md

```diff
@@ -23,19 +23,15 @@ Here is a possible program to test your function :
 ```go
 package main
 
-import (
-	"fmt"
-
-	"piscine"
-)
+import "fmt"
 
 func main(){
     a := []string{"coding", "algorithm", "ascii", "package", "golang"}
-    fmt.Printf("%#v\n", piscine.Slice(a, 1))
-    fmt.Printf("%#v\n", piscine.Slice(a, 2, 4))
-    fmt.Printf("%#v\n", piscine.Slice(a, -3))
-    fmt.Printf("%#v\n", piscine.Slice(a, -2, -1))
-    fmt.Printf("%#v\n", piscine.Slice(a, 2, 0))
+    fmt.Printf("%#v\n", Slice(a, 1))
```

Comment:
Like before, may I ask why this one was modified instead of creating a new one with a new ref like the rest?

If it doesn't require any action, feel free to resolve the conversation slightly_smiling_face

Reply:
same thing here